### ISSUE TITLE
nothing happens after installing DTV nightly

### ISSUE THREAD

**Issue by xatuple** [SchizoDuckie/DuckieTV#1292]
I have downloaded the DTV nightly stand alone version for mac.  
after I execute the DuckieTV-202011202304-OSX-x64.pkg file, it goes trhough the installation process and I get a confirmation that it was installed.
This is on a macMini late 2012, running macOS 10.15.7

but after the installation, I can't find the icon to run DTV... I still have the version 1.1.5 and in the applications the icon that appears there to open the app is for only this version.

so basically, it did not did an update, and cant find the icon to use DTV

**Comment by garfield69** [comment]
Yeah, on the mac, the Nightly installer will silently fail if the old app is still there.
You first need to move the existing DuckieTV app from your Applications folder to the bin, and then run the Nightly installer.

**Comment by xatuple** [comment]
that did the trick !! thank you
On Fri, Dec 18, 2020 at 11:48 AM, garfield69 <notifications@github.com> wrote:


Yeah, on the mac, the Nightly installer will silently fail if the old app is still there.
You first need to move the existing DuckieTV app from your Applications folder to the bin, and then run the Nightly installer.

**Comment by xatuple** [comment]
> Yeah, on the mac, the Nightly installer will silently fail if the old app is still there.
> You first need to move the existing DuckieTV app from your Applications folder to the bin, and then run the Nightly installer.

that worked !! thank you.
